python/tvm/relay/op/contrib/dnnl.py

```diff
@@ -63,3 +65,23 @@ def _func_wrapper(attrs, args):
 _register_external_op_helper("add")
 _register_external_op_helper("subtract")
 _register_external_op_helper("multiply")
+
+
+def make_pattern(with_bias=True):
+    data = _expr.var("data")
+    weight = _expr.var("weight")
+    bias = _expr.var("bias")
+    conv = reg.nn.conv2d(data, weight)
```

Comment:
Why don't we import `op` as just `op`, rather than `reg`? `reg.nn.conv2d` seems very weird...